TODO/the-ultimate-guide-to-creating-a-mobile-application.md

```diff
@@ -8,191 +9,191 @@
-### Prototyping
+### 原型
 
-The original concept behind the term ‘prototype’ is the sample model of the product that gives the ability to test it and see if the solutions and decisions made about the product are efficient. Prototypes should not be seen as the analogue of the final product as they aren’t those. Their main aim is to enable a designer, a customer and a user to check the correctness and appropriateness of the design solutions.
+“原型”最初的含义是用来测试解决方案或设计是否有效的样品。原型不应该被看作最终产品的相似物，因为它们本来就不是（用来做最终产品演示的）。原型的目标是让设计师、顾客和用户来检查设计是否正确且合适。.
 
-The value of prototypes in the sphere of app and webdesign has rocketed for the last few years. Actually, it is easy to explain as even the low-fidelity prototype gets the designer, customer and tester much closer to the looks and functions of the future product than the most elaborate schemes, drawings and wireframes. Certainly, that doesn’t mean that schemes and wireframes could be eliminated from the process as they are essential in the process of creating design solutions. However, when you want to feel their efficiency and check if nothing has been missed in the design process, prototype will be the great help.
+围绕应用和网页设计的原型的价值在近几年跑的比博尔特还快。事实上这很容易解释，尽管原型清晰度很低，但相比复杂计划、图示和线框图来说，原型使设计师、顾客和测试离最终产品的外观和功能更近了。当然，这并不表示计划和线框图可以被省略，它们仍是设计中必不可少的一部分。然而，当你想感受一下性能并检查设计是否有遗漏时，原型会帮大忙的。
 
-Considering the fact that a lot of customers see the prototype as something very close to the final version of product design aka “UI in action”, in practice this approach is not effective. Prototyping is much more efficient and useful as the step between UX design and UI design. So, here in [Tubik Studio](http://tubikstudio.com/) we support the workflow having the sequence «UX — prototype — UI».
+考虑到实际上许多客户都将原型当作十分接近最终版的产品设计，或者叫做“演示UI”，这种操作在实际中效率并不高。原型是在 UX 设计和 UI 设计中更高效和有用的一步。所以我们在 [Tubik Studio](http://tubikstudio.com/) 中提供了 UX-原型-UI 步骤的工作流。
 
-The prototypes on UI stage can be created for presentation of application general looks rather than for testing and improving its functional features. And this is the trap in which it is easy to get confused. Prototyping all the details on the final stage of UI in most cases is not so reasonable as it could seem. It will be too time-consuming and in this perspective, it would be better to spend the same time on coding a demo-version. Moreover, usability should be thoroughly checked first of all at the UX stage, otherwise it would be much harder to change inefficient solution after having accomplished a lot of work on UI. Certainly, it would be amazing to create prototypes both for UX and UI, but by far not all the designers and customers agree to spend so much time on design tasks and want to test and improve the design much faster and cheaper.
+UI 阶段的原型是建立用来展示应用大概样式的而不是测试和提升功能特性。这是一个经常被搞混的点。在原型阶段就设计全部 UI 在大多数情况下看起来是没什么必要的。这太浪费时间了而且在这个阶段将时间花费在 一个 Demo 版本上会更好的。此外，可用性应该在 UX 阶段就已经完整地检查过了，否则在完成众多 UI 工作后再度调整不合适的成果会相当困难。当然，如果有既包含了 UX 又 有 UI 的原型是相当棒的，但目前为止并不是所有的设计师和顾客都同意将大量时间耗在设计阶段，他们想又快又便宜地测试和改善设计。
```

Comment:
otherwise it would be much harder to change inefficient solution after having accomplished a lot of work on UI. 

否则，在大部分 UI 工作完成后再去改变那些不够高效的用例就会非常困难。